proposals/stage-1/TRAVEL_FUND/MEMBER_TRAVEL_FUND.md

```diff
@@ -0,0 +1,169 @@
+Click [here](https://events.linuxfoundation.org/events/node-js-interactive-2018/attend/visa-request/) to request a visa letter.
+
+### What the Foundation Cannot Do:
+
+Please note that unless the Foundation is specifically hosting the event, we cannot help with visa letters.
+The Foundation is also not able to contact consulates. When the Foundation has done this in the past, our efforts have been unsuccessful
+and/or we’ve lacked the expertise to be effective.  
+
+For additional information on preparing your visa application or requesting a visa letter, please email
+[visainfo@openjsf.org](mailto:visainfo@openjsf.org).
+
+## Requesting Travel Funds
+
+* Open a pull request that adds an entry to the appropriate year's file
+  * Include as many fields as possible:
+    * The name of the event you plan to attend.
+    * The location, dates of the event and where you will be traveling from.
+    * The presentation you intend to give, if applicable.
+    * The size of the stipend you wish to receive.
+      * In the pull request description, explain how you calculated your
+        stipend. For example, include estimates for airfare, hotel, other
+        transportation, and so on. The total amount should be your stipend
+        request.
+      * In the PR description, mention @openjs-foundation/cpc.
+    * Once the final amount spent is known, update the table again with that information.
+    * Certain details of your request (such as the column for date sent to finance) may be unknown initially; these can be filled in and updated during the process of reconciliation.
+
+## Approval
+
+A request is approved when all of the following criteria are met:
+
+* The pull request has been open for at least 72 hours.
+* The request has approval from at least four members of @openjs-foundation/cpc, including a voting representative for your project.
+* No members of @openjs-foundation/cpc have objected to the request.
+* The pull request has been merged. Any member may merge once other prerequisites have been met.
+
+If a request is blocked by an objection from a member of @openjs-foundation/cpc, then the only way for the request
+to be approved is for that committee to decide to approve the request using whatever decision-making process the committee uses when
+consensus is not reached.
+
+As examples, the purpose of prior sponsored travels includes attending TC39 meetings,
+attending [OpenJSF collaboration summits](https://github.com/nodejs/summit), and
+mentoring [Code and Learn](https://github.com/nodejs/code-and-learn).
+
+### Considerations for approval of a request
+
+#### Impact
+* Preference is given to events hosted or sponsored by the OpenJSF Foundation. For example, OpenJSF collaboration summits and Diagnostics summits should be given precedence because they bring together members of the Foundation to help the Foundation projects advance.
```

Comment:
```suggestion
* Preference is given to events hosted or sponsored by the OpenJS Foundation. For example, OpenJS Collaboration summits and Diagnostics summits should be given precedence because they bring together members of the Foundation to help the Foundation projects advance.
```